ansible/roles/openstack-ansible-os_triliovault_wlm/defaults/main.yml

```diff
@@ -181,10 +181,6 @@ triliovault_opt_dir: "/opt/triliovault-wlm"
 triliovault_virt_env: "/home/stack/wlm-venv"
 
 wlm_log_cfg: "/etc/triliovault-wlm/wlm_logging.conf"
-WLM_API_LOG_FILE: "{{ triliovault_log_dir }}/triliovault-wlm-api.log"
```

Comment:
was it needed to remove these lines ?

Reply:
the removed vars are no more in use hence, removed from the default/main.yml